build.sbt

```diff
@@ -22,6 +22,7 @@ lazy val plugin = project
   .settings(common)
   .settings(
     name := """sbt-swagger-codegen""",
+    crossSbtVersions := Seq("0.13.16", "1.0.4"),
```

Comment:
actually also `1.1.0-RC1` is out :-P

Reply:
I know... I settled for the stable version